src/main/resources/validationrules/iac/iac-case-role-validation.drl

```diff
@@ -24,7 +25,7 @@ when
              roleType == RoleType.CASE,
              roleName == "tribunal-caseworker",
              attributes["caseId"] != null && $caseId : attributes["caseId"].asText())
-         Case(
+    $c:  Case(
```

Comment:
no need to assign in $c variable.

Reply:
we need to pick values from the case instead of any hard coding in the then part.

Reply:
but you are not using this $c variable anywhere in drool file so  no meaning to define it.

Reply:
Yes, that's a miss. The jurisdiction and casetype are supposed to come from case reference. updated.